.eslintrc.json

```
@@ -1,8 +1,13 @@
 {
   "plugins": [
-    "@cypress/dev"
+    "@cypress/dev",
+    "eslint-plugin"
   ],
   "extends": [
     "plugin:@cypress/dev/general"
```

You'll want to extend the `recommended` config of this plugin:

```suggestion
    "plugin:@cypress/dev/general",
    "plugin:eslint-plugin/recommended"
```